tests/test_resource_requests.py

```diff
@@ -238,11 +238,6 @@ def test_with_column_folded_rayrunner_class():
         "more_memory_request",
         assert_resources_1(col("id"), num_cpus=1, memory=5_000_000),
     )
-    assert_resources_2 = assert_resources.override_options(num_cpus=1, memory_bytes=None)
-    df = df.with_column(
-        "more_cpu_request",
-        assert_resources_2(col("id"), num_cpus=1),
-    )
```

Comment:
removing this because it will stall due to the stateful UDFs asking for more CPUs than the mac machine has